library/cwm/src/lib/cwm/abstract_widget.rb

```diff
@@ -1,6 +1,22 @@
 require "abstract_method"
 
 module CWM
+  # A Yast::Term that can be passed as is to Yast::UI methods
+  # (OpenDialog, ReplaceWidget)
+  class UITerm < Yast::Term; end
```

Comment:
@jreidinger has in the past objected to this kind of "code for documentation  purposes only". So I tried to avoid this :) We are using `@!method` Yard macros elsewhere, but there is no `@!class`. You can use `@!parse class Foo; end` which does make the class appear... but the documentation is missing.